static/js/root.jsx

```diff
@@ -207,30 +220,145 @@ function RestaurantDetails() {
     return (
         <div className = "restaurant_details">
             <h1 id = "rest_name">{name}</h1>
             <p>Website: {website}</p>
             <p>Address: {vicinity}</p>
             <p>Hours: {hours}</p>
             <p>Google Rating: {googleRating}</p>
-            <button onClick ={()=>{return <ReadRatings/>}}>Read ratings</button>
-            <button>Rate this restaurant</button>
+            <button onClick  = {()=>{setRateRest(true)}}>Rate this restaurant</button>
+            <WriteReview/>
+            <button onClick = {()=>{setRateRest(false)}}>Back to reviews</button>
+        
         </div>
     );
+    } else {
+    return (
+        <div className = "restaurant_details">
+            <h1 id = "rest_name">{name}</h1>
+            <p>Website: {website}</p>
+            <p>Address: {vicinity}</p>
+            <p>Hours: {hours}</p>
+            <p>Google Rating: {googleRating}</p>
+            <button onClick  = {rateIt}>Rate this restaurant</button>
+            <div className = "rest_ratings"><ShowRatings restID={ID}/></div>
+        
+        </div>);
+    }
 }
-function showRatings(props) {
-    restID = props.restID
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+function ShowRatings(props) {
+    const restaurantID = {"restID": props.restID};
+    const [ratingsList, setRatingsList] = React.useState([]);
+    React.useEffect(()=> {
+        fetch('/api/get-ratings', {
+            method: 'POST',
+            body: JSON.stringify(restaurantID),
+            headers: {
+                'Content-Type': 'application/json'
+                },
+        })
+        .then(response => response.json())
+            .then(data => {
+                const ratings_array = []
+                console.log(data)
+                let i =0;
+                for (const rating of data) {
+                    console.log(i);
+                    i=i+1;
+                    console.log("rating", rating)
+                    console.log(rating["user"])
+                    console.log("User", rating.rating_id); 
+                    let outdoors;
+                    if (rating[1]["scores"][3] === "true") {
+                         outdoors = "Yes";
+                    } else {
+                         outdoors = "No";
+                    }
+                    ratings_array.push(
+                    <div key={i}>
+                        <h3>User: {rating[0]["user"][0]} {rating[0]["user"][1]}</h3>
+                        <p>Cleanliness: {rating[1]["scores"][0]}</p>
+                        <p>Masks: {rating[1]["scores"][1]}</p>
+                        <p>Social Distancing Enforced: {rating[1]["scores"][2]} </p>
+                        <p>Outdoor Seating: {outdoors} </p>
+                        <p>Comments: {rating[1]["scores"][4]}</p>
+                    </div>
+                    );
+                }
+                setRatingsList(ratings_array);
```

Comment:
as per our conversation, dont store html into `State`. 
State usually store data, so in this case, the ratings. 

and the rendering of the data should be done outside (line301).